README.md

```diff
@@ -1,503 +1,62 @@
-```
-
-You also need to enable your Nuxt application to use your identity provider as a source of users. To do this you'll use another npm package to make the application OAuth aware.
-
-In your terminal, stop the running application and use the following command to install the library:
-
-```
-npm i nuxt-oauth
 ```
 
-> Note: there is an official [Nuxt Community Auth module](https://auth.nuxtjs.org/) which supports OAuth. However, as of this writing, it does not include a nonce on requests for tokens. Therefore, it is not standards compliant.
-
-Once that is installed, you need to configure its behavior in the `nuxt.config.js`. First, add `nuxt-oauth`  to the array of modules.
-
-```
-  modules: [
-    ['nuxt-oauth']
-  ],
-```
-
-Then add a new `oauth` section to configure the OAuth library.
-
-```
-  oauth: {
-    sessionName: 'WidgetCoSession',
-    secretKey: process.env.SECRET_KEY,
-    oauthHost: process.env.OAUTH_ISSUER,
-    oauthClientID: process.env.CLIENT_ID,
-    oauthClientSecret: process.env.CLIENT_SECRET,
-    scopes: ['openid', 'profile'],
-  },
-```
-
-This implementation requires a Vuex store. Nuxt will not create the store by default, you must create an empty `index.vue` file in the `store` folder. Once Nuxt detects this file it will include the correct dependencies for you.
-
-Now that you have OAuth configured, you need to configure which routes require authentication. You can do this by adding the value `authenticated: true` to the default export of a page. Update the script tag of `progress.vue` in the `pages` folder with the following code.
-
-```
-<script>
-const axios = require('axios'); 
-export default {
-  authenticated: true,
-  asyncData ({ params }) {
-    return axios.get('http://localhost:3000/status.json')
-    .then((res) => {
-      return {
-        statusUpdate: res.data.statusUpdate,
-        statusDate: res.data.statusDate
-      }
-    })
-  }
-}
-</script>
-```
-
-Launch a private session in your browser and revisit localhost:3000. Now when you navigate to the progress screen you’ll be required to login.
-
-![End to end demo](./images/demo.gif)
+You can now use the sample, when you navigate to the ```/progress``` route you will be redirected to Okta to sign in.
 
-Congratulations! You've now built a universal JavaScript application! Content is prepared on the server and sent to the user as multiple SEO-friendly pages complete with route specific metadata. You have used `asyncData` to populate your pages with dynamic content from an API. Finally you have protected sensitive information behind an identity provider and can force users to login before granting access.
+## Help
 
-## Learn More About Vue and Authentication
+Please post any questions as comments on the [blog post](https://developer.okta.com/blog/2019/04/26/tutorial-build-universal-apps-with-nuxt), or visit our [Okta Developer Forums](https://devforum.okta.com/). You can also email developers@okta.com if would like to create a support ticket.
```

Comment:
```suggestion
Please post any questions as comments on the [blog post](https://developer.okta.com/blog/2019/04/26/tutorial-build-universal-apps-with-nuxt), or visit our [Okta Developer Forums](https://devforum.okta.com/). 
```

We're starting to remove developers@okta.com in favor of the forums.

Reply:
@andymarch If I try to run this locally, I get a 400 on due redirect_uri. How do we avoid that? I know there's an article about it in the knowledgebase but it doesn't seem to work.